Documentation/CRDs/ceph-nfs-crd.md

```diff
@@ -204,3 +204,17 @@ the size of the cluster.
     ceph orch set backend ""
     ceph mgr module disable rook
     ```
+
+## External Clusters
+
+User Applications can make use of the NFS client running on the external ceph standalone cluster.
+
+Procedure:
+
+Export the nfs client to a particular cephfs filesystem:
+
```

Comment:
I'm not sure how to best put this into words, but I have seen this pattern in our docs more than once, and I'm always a little confused by it.

Generally, we don't need to use a colon (`:`) at the end of a sentence unless it immediately precedes and introduces:
- a code block
- a list
- a block quote

The above is a prime example.

In this case "Procedure:" seems like maybe it should just be a section header (which shouldn't end in a colon). Or maybe it can be omitted entirely since the following line does a better job of explaining the subsequent code block.